### ISSUE TITLE
There is no Oregen on TcetiE

### ISSUE THREAD

**Issue by boubou19** [GTNewHorizons/NewHorizons#2670]
#### Which modpack version are you using?
2.0.2.7
#
#### If in multiplayer; On which server does this happen? 
delta
#
#### What did you try to do, and what did you expect to happen?
i expected to encounter at least one vein on Tceti E with the zpm ore scanner
#
#### What happend instead? (Attach screenshots if needed)
i found none

**Comment by bartimaeusnek** [comment]
addition:
the OreScanner does not even open its GUI there.

**Comment by bartimaeusnek** [comment]
@richardhendricks can you have a look please?

**Comment by Drobac** [comment]
Cant be spawned, same as europa and few others

**Comment by boubou19** [comment]
so why some veins are listed to spawn on Tceti E in the ore gen sheet available in the help channel? O_o

**Comment by Dream-Master** [comment]
There are a few Planets that cant spawn gt ores. The Game stuck so i let the planets open.
Schame on Galaxy Space code.

**Comment by MineAnPlay** [comment]
I listed what ever was listed in the configs (unless I accidentally made an X).

**Comment by Dream-Master** [comment]
There are no ores on Tceti and Europa. I spend houres with namikon to fix Bullshit code from Galaxy space but without any success. Maybe we update GS one day and see if this was fixed but for now no ores on Europa and TCeti. Look into config there are no ores. And if there ores sted to true it not worked for sure.

**Comment by Dream-Master** [comment]
https://github.com/GTNewHorizons/NewHorizonsCoreMod/blob/master/src/main/java/com/dreammaster/galacticgreg/SpaceDimRegisterer.java#L193
https://github.com/GTNewHorizons/NewHorizonsCoreMod/blob/master/src/main/java/com/dreammaster/galacticgreg/SpaceDimRegisterer.java#L199
https://github.com/GTNewHorizons/NewHorizonsCoreMod/blob/master/src/main/java/com/dreammaster/galacticgreg/SpaceDimRegisterer.java#L204

**Comment by richardhendricks** [comment]
I assume you also added the dimensions to the dimension spawn list?

https://github.com/GTNewHorizons/GalacticGregGT5/blob/a0ac9e63dbfff3c0c9de7e16588206cab8931932/src/main/java/bloodasp/galacticgreg/ModRegisterer.java#L167

**Comment by richardhendricks** [comment]
Ah, nevermind, I see you added it to the coremod.  I need to spend some time looking at both this weekend.

**Comment by richardhendricks** [comment]
(notes for me)
https://github.com/GTNewHorizons/NewHorizons/blob/master/config/GalaxySpace/core.conf
https://github.com/GTNewHorizons/NewHorizons/blob/master/config/GalaxySpace/dimensions.conf
https://github.com/GTNewHorizons/NewHorizonsCoreMod/blob/master/src/main/java/com/dreammaster/galacticgreg/SpaceDimRegisterer.java

**Comment by richardhendricks** [comment]
I have rootcaused the issue with Galaxyspace as extreme worldgen cascading.  With no galacticgreg installed, GS decorations cause severe over-chunks.

https://github.com/BlesseNtumble/GalaxySpace/issues/214

I can try putting in a per-chunk oregen I did for GT5U into GG, but it wouldn't stop the cascading caused by the mod itself.

Also, the worldgeneration.cfg file does not use the correct name for the dimension, I will submit pull request.
